### DISCUSSION TITLE
Adding DrawingView not possible

### DISCUSSION THREAD

**Discussion by LukasHeintze-LabelSoftware** [unoplatform/uno#14410] [Q&A]
Hello, I have done all the steps described in the documentation under https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/Maui/ThirdParty-MauiCommunityToolkit.html#adding-drawingview. 
The project starts under the Windows platform. But the moment the xaml is loaded, it crashes. Can anyone help me?

![image](https://github.com/unoplatform/uno/assets/115161457/b178f7c3-cb6d-4f94-8c18-e3ff31cf135f)
Message of the Exception = "The type initializer for 'WinRT.ActivationFactory`1' threw an exception."
Looks like something internal, doesn't it?

**Comment by jeromelaban** [comment]
Have seen this in the past @dansiegel ?

**Reply by dansiegel** [reply]
It's hard to say what could be causing this or what the potential fix is though unless I have a reproduction of the issue. @LukasHeintze-LabelSoftware can you post a sample that reproduces the issue.

**Reply by Hurricane31337** [reply]
Here is a sample project that has this issue on latest .NET 8.0 (on Windows 10.0.19045 Build 19045) at least. You need to build the app with the separate Uno5Net8Test.Windows.csproj and in unpackaged mode (I used x64).

[Uno5Net8Test.zip](https://github.com/unoplatform/uno/files/13404886/Uno5Net8Test.zip)
